src/librustc/metadata/encoder.rs

```diff
@@ -404,8 +430,21 @@ fn encode_reexported_static_methods(ecx: &EncodeContext,
                                     rbml_w: &mut Encoder,
                                     mod_path: PathElems,
                                     exp: &def::Export) {
-    if let Some(ast_map::NodeItem(item)) = ecx.tcx.map.find(exp.def_id.node) {
-        let path_differs = ecx.tcx.map.with_path(exp.def_id.node, |path| {
+    let exp_node_id = if let Some(n) = ecx.tcx.map.as_local_node_id(exp.def_id) {
+        n
+    } else {
+        // Before the refactor that introducd `as_local_node_id`, we
+        // were just extracting the node and checking into the
+        // ast-map. Since the node might have been from another crate,
+        // this was a tenuous thing to do at best. Anyway, I'm not
+        // 100% clear on why it's ok to ignore things from other
+        // crates, but it seems to be what we were doing pre-refactor.
+        // -nmatsakis
```

Comment:
AFAICT this function is useless atm and can be removed wholesale (there's a few more like it).